tests/es_client.py

```diff
@@ -0,0 +1,46 @@
+import json
+import argparse
+import elasticsearch
+from esFrontLine.connection import HawkConnection
+
+
+def main(settings):
+
+    # Load flask config
+    flask = settings.get('flask')
+    assert flask and 'host' in flask and 'port' in flask, \
+        'Invalid flask settings'
+    server_url = 'http://{host}:{port}'.format(**flask)
+    print('Will connect to esfrontline: {}'.format(server_url))
+
+    # Load users config
+    users = settings.get('users')
+    assert users, 'Missing users in settings'
+    user = users[0]
+    assert 'hawk' in user, 'Missing hawk conf in first user'
+    assert 'resources' in user, 'Missing resources in first user'
+    print('Will connect as {}'.format(user['hawk']['id']))
+
+    es = elasticsearch.Elasticsearch(
+        hosts=[server_url],
+        connection_class=HawkConnection,
```

Comment:
your `HawkConnection` solution is beautiful

Reply:
Thanks !